tests/unit/local/apigw/test_path_converter.py

```diff
@@ -19,6 +19,13 @@ def test_proxy_path(self):
 
         self.assertEquals(flask_path, "/<path:proxy>")
 
+    def test_proxy_path_with_different_name(self):
```

Comment:
Can you also test the following cases (mostly testing combination of single & multiple path params with wildcard

1. `/a/{id}/b/{resource+}`
2. `/a/b/{proxy}/{resource+}`
3. `/{id}/{something+}`
4. `/{a}/{b}/{c}/{d+}`

Reply:
Sure can do :)